webui/pom.xml

```diff
@@ -127,6 +127,21 @@
               </arguments>
             </configuration>
           </execution>
+          <execution>
+            <id>run-linter</id>
+            <phase>test</phase>
```

Comment:
should this be a part of the test phase? Do you think it's more apt to put it in `checkstyle` or `verify`?

 Will `eslint` fix the issues? or just complain about them?

Reply:
from what i tried, i couldn't put it in `checkstyle` because i need all the extra steps of installing node/npm and running `npm install` first to get lerna ready. i'm not familiar with maven so let me know if if i got it all wrong.

i moved it to `verify` phase.

`eslint` will complain about the issues. dev will need to fix them with running `npm run format` or manually fix.

Reply:
makes sense, thanks!